src/sentry/static/sentry/app/components/stream/group.jsx

```diff
@@ -93,19 +104,62 @@ const StreamGroup = createReactClass({
     SelectedGroupStore.toggleSelect(this.state.data.id);
   },
 
+  toggleShowLifetimeStats(showLifetimeStats) {
+    if (this.hoverWait) {
+      clearTimeout(this.hoverWait);
+    }
+
+    this.hoverWait = setTimeout(() => this.setState({showLifetimeStats}), 100);
+
+    this.setState({showLifetimeStats});
+    this.forceUpdate();
```

Comment:
Why the `forceUpdate` if we are updating state?

Reply:
Otherwise on hover after toggling `showLifetimeStats`, the view was not re-rendering

Reply:
Should use `shouldComponentUpdate`, I haven't thought deeply about this